### PR TITLE
Remove hard-coded jar name from docker script

### PR DESCRIPTION
**Pull request by anusudarsan**
@kokosing can you review please?

### CODE REVIEW

File: presto-yarn-test/bin/run_on_docker.sh [hunk 1]
```diff
@@ -33,8 +33,9 @@ function check_hive() {
 }
 
 function run_product_tests() {
+  MVN_VERSION=$(mvn -q -Dexec.executable="echo" -Dexec.args='${project.version}' --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
```
Comment (kokosing) [inline presto-yarn-test/bin/run_on_docker.sh]: In `presto-product-tests` I did it differently. See https://github.com/prestodb/presto/blob/master/presto-product-tests/bin/run_on_docker.sh#L146
and https://github.com/prestodb/presto/blob/master/presto-product-tests/src/main/resources/presto.env
and https://github.com/prestodb/presto/blob/master/presto-product-tests/pom.xml#L108

File: presto-yarn-test/bin/run_on_docker.sh [hunk 2]
```diff
@@ -33,8 +33,9 @@ function check_hive() {
...
   compose run runner \
-    java -jar /workspace/target/presto-yarn-test-1.2.1-executable.jar \
+    java -jar /workspace/target/presto-yarn-test-$MVN_VERSION-executable.jar \
```
Comment (kokosing) [inline presto-yarn-test/bin/run_on_docker.sh]: it is not a version of maven but it is version of presto-yarn. So I would name it PRESTO_YARN_VERSION

**Comment by kokosing** [comment]
looks good % variable name. Optionally you can implement this via resource filtering in maven, see my other comment.

**Comment by anusudarsan** [comment]
renamed. Thanks for the maven tip (maybe I can update it to use resource filtering if the current version creates any issues).

**Pull request merged**